### PR TITLE
setup test

### PR DESCRIPTION
**Pull request by dev-romelus**

### CODE REVIEW

File: src/components/ui/Count.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable react/jsx-no-bind */
+import { useState } from 'react';
```
Comment (dev-romelus) [inline src/components/ui/Count.tsx:2]: Remove this file.

File: src/context/actions/auth.action.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { ILogin, AuthDispatch } from '../../types/index';
```
Comment (dev-romelus) [inline src/context/actions/auth.action.ts:1]: Remove this file.

File: src/context/reducers/auth.reducer.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import { Action, IAuthState } from '../../types';
```
Comment (dev-romelus) [inline src/context/reducers/auth.reducer.ts:1]: Remove this file.

File: src/context/reducers/index.ts [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import { Action, State } from '../../types';
+import authReducer from './auth.reducer';
+import userReducer from './user.reducer';
+
+export default (states: State, action: Action) => ({
+    authState: authReducer(states.authState, action),
```
Comment (dev-romelus) [inline src/context/reducers/index.ts:6]: remove this line.

File: src/context/states/index.ts [hunk 5]
```diff
@@ -0,0 +1,5 @@
+export const initialState = {
+    authState: { isAuthenticated: false, isLoading: false },
```
Comment (dev-romelus) [inline src/context/states/index.ts:2]: remove this line.

File: src/context/types/auth.type.ts [hunk 6]
```diff
@@ -0,0 +1,9 @@
+export enum AuthType {
```
Comment (dev-romelus) [inline src/context/types/auth.type.ts:1]: Remove this file.

File: src/context/types/notification.type.ts [hunk 7]
```diff
@@ -0,0 +1,4 @@
+export enum NotificationType {
```
Comment (dev-romelus) [inline src/context/types/notification.type.ts:1]: Remove this file.

File: src/hooks/usePublicRoute.ts [hunk 8]
```diff
@@ -0,0 +1,13 @@
+import { useEffect } from 'react';
```
Comment (dev-romelus) [inline src/hooks/usePublicRoute.ts:1]: remove this file.

File: src/utils/enums/socket.enum.ts [hunk 9]
```diff
@@ -0,0 +1,6 @@
+export enum SocketEvents {
```
Comment (dev-romelus) [inline src/utils/enums/socket.enum.ts:1]: remove SEND_NOTIFICATION. add CREATE_NOTIFICATION and GET_NOTIFICATIONS

**Review by app-robuste — CHANGES_REQUESTED** [review]
Cf comments.

**Review by app-robuste — APPROVED** [review]

**Pull request merged**